x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/alerts_tab_badge.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiToolTip } from '@elastic/eui';
+import { EuiNotificationBadge } from '@elastic/eui';
+import { EuiIcon } from '@elastic/eui';
+import { EuiLoadingSpinner } from '@elastic/eui';
```

Comment:
Could we import them in a single import?

```suggestion
import { EuiToolTip, EuiNotificationBadge, EuiIcon, EuiLoadingSpinner } from '@elastic/eui';
```